rasa/core/tracker_store.py

```diff
@@ -527,9 +549,12 @@ def retrieve(self, sender_id):
         """
         stored = self.conversations.find_one({"sender_id": sender_id})
 
+        if not stored:
+            return
```

Comment:
I don't think we can return here if we want to maintain the `isdigit` case

Reply:
🙈